### PR TITLE
Stop order endpoints

### PR DESCRIPTION
**Pull request by keyganker**

### CODE REVIEW

**Comment by keyganker** [comment]
@dexter-2 we'll release it on 4.9

File: locales/en.yml [hunk 1]
```diff
@@ -104,6 +104,10 @@ en:
   changelog_new: new
   changelog_update: update
   changelog_bugfix: bugfix
+  ## 2020-04-10
+  bugfix_20200410: Change market api curl example URL
```
Comment (dextertd) [inline locales/en.yml:108]: This doesn't need to be displayed in the changelog with `t(:bugfix_20200410)` as users don't need to know about changes to the docs.

File: locales/en.yml [hunk 2]
```diff
@@ -104,6 +104,10 @@ en:
...
+  # just fix chinese part doc, so english changelog is empty
+  bugfix1_20200409: refine chinese doc
```
Comment (dextertd) [inline locales/en.yml:110]: Same here (see above comment)

File: source/includes/linear_future/_account_data.md [hunk 3]
```diff
@@ -322,6 +321,7 @@ POST
 |<a href="#trigger-price-type-trigger_by">trigger_by</a> | false | string | t(:row_comment_triggerBy)|
 |close_on_trigger |false |bool |t(:row_comment_closeOnTrigger)
 |order_link_id |false |string |t(:row_comment_orderLinkId) |
+|reduce_only |true |bool |t(:linear_row_comment_reduceOnly) |
```
Comment (dextertd) [inline source/includes/linear_future/_account_data.md:324]: Should this be `true`? I notice `reduce_only` and `close_on_trigger` are also set for true in linear Place Active Order. I'm not sure if this is correct?

**Review by dextertd — CHANGES_REQUESTED** [review]
#35

File: source/includes/linear_future/_account_data.md (hunk 3 above)
Comment (keyganker) [reply source/includes/linear_future/_account_data.md:324]: Will mark `close_on_trigger ` as required  to be consistent with ` linear Place Active Order`.
Thanks!

**Pull request merged**